src/h5web/vis-packs/core/shared/AxisSystem.module.css

```diff
@@ -22,37 +23,24 @@
 
 .axis[data-type='abscissa'] {
   grid-area: bottom-axis;
-  text-anchor: middle;
 }
 
 .axis[data-type='ordinate'] {
-  grid-area: axis-grid; /* `AxisLeft` aligns ticks to the outer left of its own bounding box  */
```

Comment:
The comment was on the wrong line, but the `text-anchor` fix is no longer needed because `AxisLeft` now seems to set the correct `text-anchor` by default.